eng-paths/sr_software_engineer_3.md

```diff
@@ -0,0 +1,381 @@
+---
+title: Sr Software Engineer 3
+icon: fas fa-users
+order: 8
+layout: page-eng-paths
+---
+
+# Sr Software Engineer 3 (Sr Individual Contributor Path)
+### Years of xp (guideline)
+<table  markdown="1">
+<tr>
+<td>
+
+Sr Software Engineer 2
+
+</td>
+<td style="font-weight:bold">
+
+Sr Software Engineer 3
+
+</td>
+<td>
+
+Staff Engineer
+
+</td>
+</tr>
+<tr>
+<td  markdown="1">
+
+7+
+
+</td>
+<td  markdown="1" style="font-weight:bold">
+
+9+
+
+</td>
+<td  markdown="1">
+
+11+
+
+</td>
+</tr>
+</table>
+
+
+### Scope of action
+<table  markdown="1">
+<tr>
+<td>
+
+Sr Software Engineer 2
+
+</td>
+<td style="font-weight:bold">
+
+Sr Software Engineer 3
+
+</td>
+<td>
+
+Staff Engineer
+
+</td>
+</tr>
+<tr>
+<td  markdown="1">
+
+2-3 Products / Key components
+
+</td>
+<td  markdown="1" style="font-weight:bold">
+
+- Multiple Products/Key Components.
+- All team's domain and tasks that involve cross-teams work.
+
+</td>
+<td  markdown="1">
+
+Whole platform
+
+</td>
+</tr>
+</table>
+
+
+### Scope of knowledge (Breadth and Depth)
+<table  markdown="1">
+<tr><td>
+
+Sr Software Engineer 2
+
+</td></tr>
+<tr><td  markdown="1">
+
+- Technical expert on 2 products / key components, solid on the rest of the platform.
+- Expert in primary focus area (BE/FE/Data/or DevOps).
+- Strong proficiency in DevOps, PG/CH, Observability, and other platform technologies.
+
+</td></tr>
+<tr><td style="font-weight:bold">
+
+Sr Software Engineer 3
+
+</td></tr>
+<tr><td markdown="1" style="font-weight:bold">
+
+- Technical expert on multiple products / key components, knowing all the extent of the team's domain.
```

Comment:
Similar comment here for the "knowing all the extent of the team's domain" portion of the sentence. Are you referring to all teams or the individual's core team? (if the former, then 'extent of all teams' domains' if the latter then 'extent of their team's domain')